New name & first exercise (#44)
CodeReadingClubs/www: content/blog/countdown-3.md

---
{
  "type": "blog",
  "author": "Felienne Hermans",
  "title": "Code Reading Club Reading Countdown - 3",
  "description": "Code Reading Club Reading Countdown - 3",
  "image": "images/articles/CRCRC-3.png",
  "draft": true,
  "published": "2021-12-03",
}
---

## Countdown - Snippet 3

Today's code reading countdown.

**Snippet**

![CRCRC-3](/images/articles/CRCRC-3.png)

**Question**

> Which JavaScript behavior(s) and feature(s) does t his code exploit to *not* break where other languages might protest?

Reviewer: ```suggestion
Which JavaScript behavior(s) and feature(s) does this code exploit to *not* break where other languages might protest?
```
